pom.xml

```diff
@@ -12,7 +12,7 @@
 
 	<groupId>org.cbioportal</groupId>
 	<artifactId>cbioportal</artifactId>
-	<version>6.0.1-SNAPSHOT</version>
+	<version>6.0.0</version>
```

Comment:
is this intetnional?

Reply:
it should actually be 6.0.0-SNAPSHOT I will update now